### PR TITLE
WIFI-2096 Add Portal support for cell size management

### PR DESCRIPTION
**Pull request by lynnshi-design**

### CODE REVIEW

File: opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbRrmConfig.java [hunk 1]
```diff
@@ -216,6 +212,16 @@ AutoOrManualValue getSourcedValue(SourceType source, int profileValue, int equip
         }
         return AutoOrManualValue.createManualInstance(equipmentValue);
     }
+    
+    //For cell size related attributes, the “manual" mode will not be supported any more,
```
Comment (christieunitt-cu) [inline opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbRrmConfig.java:216]: Please change "will not be supported any more" to "is not supported"

File: opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbRrmConfig.java [hunk 2]
```diff
@@ -149,19 +145,19 @@ void configureWifiRrm(OvsdbClient ovsdbClient, String freqBand, int backupChanne
         if (multicastRate == null || multicastRate == MulticastRate.auto) {
             updateColumns.put("mcast_rate", new Atom<>(0));
         } else {
-            updateColumns.put("mcast_rate", new Atom<>(managementRate.getId()));
+            updateColumns.put("mcast_rate", new Atom<>(multicastRate.getId()));
```
Comment (christieunitt-cu) [inline opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbRrmConfig.java:148]: Nice catch!

**Review by christieunitt-cu — DISMISSED** [review]

File: opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbRrmConfig.java (hunk 2 above)
Comment (lynnshi-design) [reply opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbRrmConfig.java:148]: Thanks @rahulsharma2020 finding.

**Review by mikehansen1970 — APPROVED** [review]
approved, dependent on the cloud service changes going in first

**Pull request merged**